packages/connectors/connector-qq/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@logto/connector-qq",
+  "version": "1.0.0",
```

Comment:
use 0.1.0 or 0.0.0 as version number since the version number will automatically bump along with changeset file.